app/assets/stylesheets/cart.css.scss

```diff
@@ -88,3 +88,23 @@
   a:visited {color: #008000; text-decoration: underline; }
 }
 
+input[type="submit"].cart-item-quantity-button {
+  border : solid 0px #ffffff;
+  border-radius : .95em;
+  moz-border-radius : 3px;
```

Comment:
Unknown property moz-border-radius<br>Property name should be immediately followed by a colon